openedx/adg/lms/applications/models.py

```diff
@@ -353,6 +353,10 @@ def get_user_enrolled_course(self, user):
     def does_course_exist(self, multilingual_course):
         return self.multilingual_courses.filter(course=multilingual_course.course).exists()
 
+    @property
+    def open_multilingual_courses(self):
+        return self.multilingual_courses(manager='open_multilingual_courses').all()  # pylint: disable=no-member
```

Comment:
Actually I have done some refactoring in LP-2523 related [PR](https://github.com/OmnipreneurshipAcademy/edx-platform/pull/183). You might have to reconsider this.